rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import {terser} from 'rollup-plugin-terser';
+
+let plugins = [
+
+    // so Rollup can find externals
+    resolve({extensions: ['.js'], preferBuiltins: true}),
+
+    // so Rollup can convert externals to an ES module
+    commonjs(),
+];
+
+// minify only in production mode
+if (process.env.NODE_ENV === 'production') {
+    plugins.push(
+        // minify
+        terser({
```

Comment:
how about importing this function with the name `minify` and removing the comment on the previous line?

Reply:
It is better to keep the plugin name.

Reply:
Why exactly?

`terser` is meaningless to me, `minify` is not. If I want to know what we're using to minify I can just look at where minify is imported and I see it's terser.

Reply:
There are many plugins that do minification. It is common to keep the original name of the plugin in case people want to apply changes.

See https://github.com/TrySound/rollup-plugin-terser#usage

Reply:
I'm fine leaving it like this, I'm just suggesting something that I would argue is better in the long run (regardless of what is "common", also I don't think this would confuse anyone)